src/pages/games.tsx

```diff
@@ -0,0 +1,276 @@
+import { Footer } from "@/components/Navigation/Footer";
+import Header from "@/components/Navigation/Header";
+import { userDataSchema } from "@/components/ProfileModal/ProfileModal";
+import { useSession } from "next-auth/react";
+import { useEffect, useState } from "react";
+import { z } from "zod";
+import {
+  ChakraProvider,
+  Divider,
+  InputGroup,
+  InputLeftElement,
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  useDisclosure,
+  Text,
+  Button,
+} from "@chakra-ui/react";
+import { gameSchema, themeSchema } from "@/utils/types";
+import {
+  Search2Icon,
+  TriangleDownIcon,
+  TriangleUpIcon,
+} from "@chakra-ui/icons";
+import { Input } from "@chakra-ui/react";
+import FilterBody from "@/components/GameComponent/FilterBody";
+import chakraTheme from "@/styles/chakraTheme";
+import { useRouter } from "next/router";
+import ThemeSidebar from "@/components/GameComponent/ThemeSidebar";
+import SelectedFilters from "@/components/GameComponent/SelectedFilters";
+import GameCardView from "@/components/GameComponent/GameCardView";
+
+export default function Games() {
+  const { data: session } = useSession();
+  const currentUser = session?.user;
+  const [userData, setUserData] = useState<z.infer<typeof userDataSchema>>();
+  const [games, setGames] = useState<z.infer<typeof gameSchema>[]>([]);
+  const [themes, setThemes] = useState<string[]>([]);
+  const [selectedTheme, setSelectedTheme] = useState("All Games");
+  const [gameBuilds, setGameBuilds] = useState<string[]>([]);
+  const [gameContent, setGameContent] = useState<string[]>([]);
+  const [accessibility, setAccessibility] = useState<string[]>([]);
+  const [tags, setTags] = useState<string[]>([]);
+  const [name, setName] = useState("");
+  const router = useRouter();
+  const [empty, setEmpty] = useState(false);
+  const [filtersApplied, setFiltersApplied] = useState(false);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+
+  useEffect(() => {
+    getGames();
+    getThemes();
+  }, []);
+
+  useEffect(() => {
+    if (filtersApplied) {
+      getGames();
+      setFiltersApplied(false);
+    }
+  }, [filtersApplied]);
+
+  async function getGames() {
+    const params = new URLSearchParams();
+    params.append("page", "1");
+
+    if (gameBuilds.length > 0) {
+      gameBuilds.forEach((gb) => {
+        if (gb === "Amazon") {
+          params.append("gameBuilds", "amazon");
+        } else if (gb === "Android") {
+          params.append("gameBuilds", "android");
+        } else if (gb === "App Store") {
+          params.append("gameBuilds", "appstore");
+        } else if (gb === "Linux") {
+          params.append("gameBuilds", "linux");
+        } else if (gb === "Mac") {
+          params.append("gameBuilds", "mac");
+        } else if (gb === "WebGL") {
+          params.append("gameBuilds", "webgl");
+        } else {
+          params.append("gameBuilds", "windows");
+        }
+      });
+    }
```

Comment:
Could you try something like this instead? you might have to mess around w the syntax a little. same for game content

```
const gameBuildsMap = {
  Amazon: "amazon",
  Android: "android",
  "App Store": "appstore",
  Linux: "linux",
  Mac: "mac",
  WebGL: "webgl",
  Windows: "windows",
};

gameBuilds.forEach((gb) => {
  params.append("gameBuilds", gameBuildsMap[gb]);
});
```